### DISCUSSION TITLE
To do list / Roadmap for ReCT development?

### DISCUSSION THREAD

**Discussion by hrszpuk** [ReCT-Lang/ReCT-Go-Compiler#33] [Q&A] [answered]
Is there a list of things that need to be done, bugs that need fixing, or other contributor-related material?

**Comment by hrszpuk** [comment]
I just want a big list 💭

**Comment by mr-cheffy** [comment]
lots, but this retarded guy decided to make smth called `rectx` or smth.

**Comment by RedCubeDev-ByteSpace** [comment, marked as answer]
> Is there a list of things that need to be done, bugs that need fixing, or other contributor-related material?

no
![salami](https://user-images.githubusercontent.com/61469501/217373766-e5081778-2a9d-4809-b54d-f214c88925a9.png)

**Reply by RedCubeDev-ByteSpace** [reply]
the todo list lives in my alzheimer's brain

**Reply by hrszpuk** [reply]
😔
